components/task/details/description/Description.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+const taskFields = [
+  {
+    title: 'Akcje na stronie:',
+    listItems: [
+      "Toggle button 'Arrows' - pokazuje bądź ukrywa strzałki nawigacyjne karuzeli. (Niewidoczny w najmniejszej rozdzielczości).",
+      'Kliknięcie w strzałki przesuwa obrazy odpowiednio w lewo lub prawo.',
+      'Karuzelę można przewijać w nieskończoność (tzw. infinite carousel).',
+      'Przyciski pod karuzelą są przypisane na stałe do kolejnych obrazów i robią się aktywne (ciemno-niebieskie) w momencie gdy odpowiedni obraz znajduje się w środku karuzeli.',
+      'Środkowy obraz jest zawsze powiększony.',
+      'Klik w obraz inny niż środkowy - ustawia go w środku karuzeli.',
+    ],
+  },
+  {
+    title: 'Nice to have (poziom zaawansowany):',
+    listItems: [
+      'Kilk w obraz środkowy wyświetla go na warstwie. Obraz główny jest lekko powięszony, pozostałe proporcjonalnie zmniejszone. Zamknięcie modala - po kliknięciu drugi raz w dowolny obszar strony.',
+      'Zmiany obrazów są animowane.',
+      'Karuzela jest w pełni "dostępna", może być obsługiwana tylko za pomocą klawiatury.',
+      'Projekt posiada 100% code coverage.',
+    ],
+  },
+  {
+    title: 'Uwagi:',
+    listItems: [
+      'Nie korzystaj z frameworków CSS (bootstrap etc.) ani JS (react, angular, vue etc.)',
+      'Makiety są tylko poglądowe, odwzorowanie nie musi być pixel-perfect.',
+      'Użyty font to Poppins.',
+    ],
+  },
+];
```

Comment:
a Ty to zahardcodowałeś? :D a ja już myślałem że tak ładnie to wystylowałeś.
No dobra, nie ma tak prosto.

Dostaniesz taskDescription w postaci stinga z HTML-ką.
I to trzeba ostylować (bez customowych styli się nie obędzie uprzedzam)
na zasadzie:
.taskDescription {
& p {
  paddling-left: 8px;
}
& li {}
& ul {}
}
```
itd.

Reply:
No właśnie - czy nie lepiej taką formę ustrukturyzować - dać userowi szablon, który będzie po prostu wypełniał?

Reply:
Zrobione - najwyżej potem dopiszę jeszcze style dla części znaczników. To co w przykładowym tasku + blok kodu zostało zrobione.